docs/jetting.md

```diff
@@ -0,0 +1,760 @@
+# How to write a jet
+
+A guide for new Urbit developers.
+
+Problem statement: coder A has written some hoon, and you - coder B - need to jet it.
+
+Preconditions:
+
+* You have a pill `jetted.pill`, located in ~/tlon.
+* You have an example of hoon code that, executed in dojo, does the right thing.
+
+## Get all the source code you need.
+
+The source code you will need to modify lives in two different repos:
+urbit and arvo.  The urbit repo has the source code for the
+intrepretter / personal server, written in C.  The arvo repo has the
+operating system, written in hoon.  Since you're jetting the hoon, you
+need to modify both.
+
+For simplicitly, I'm going to assume that you are doing all of your
+development in ~/tlon.  This saves the bother of writing nonsense like
+the/path/to/your/sandbox, and also means that one can copy-and-paste
+these instructions straight into a terminal with zero editing.  Do
+your development elsewhere, if you like, and make a symbolic link.
+
+1. mkdir ~/tlon
+1. cd ~/tlon
+1. git clone https://github.com/urbit/urbit
+1. cd urbit
+1. git fetch
+1. git checkout <branch> # optional
+1. scripts/bootstrap
+1. 
+1. cd ..
+1. git clone https://github.com/urbit/arvo
+1. cd arvo
+1. git fetch
+1. git checkout <branch> # optional
+1. cd ..
```

Comment:
I understand how markdown works, but should the lists be numbered sequentially so the plaintext is more readable?

Reply:
It's in the process of being converted to Urbit markdown (or Udon or whatever we're calling it) which uses `-` for unnumbered lists and `+` for numbered lists.  I'm afraid this won't help make the plaintext more readable.